components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/config/APIKey.java

```diff
@@ -17,28 +17,29 @@
  */
 package org.wso2.apimgt.gateway.cli.model.config;
 
+import io.swagger.v3.oas.models.security.SecurityScheme.In;
+
 /**
- * Definition of basic authentication passed in to mustache
+ * Definition of APIKey passed in to mustache
  * templates.
  */
-public class BasicAuth {
-
-    private boolean isOptional;
-    private boolean isRequired;
+public class APIKey {
+    private In in;
```

Comment:
better to explain what is 'in' here in comments or use a descriptive name.

Reply:
It is the standard type "In" in swagger. https://swagger.io/docs/specification/authentication/api-keys/